Libplanet/Net/Swarm.cs

```diff
@@ -1897,13 +1897,16 @@ await foreach (Block<T> block in blocks)
 
                     receivedBlockCount += receivedBlockCountCurrentLoop;
 
-                    // FIXME: Need test
                     if (receivedBlockCountCurrentLoop < FindNextHashesChunkSize)
                     {
                         _logger.Debug(
-                            $"Got all blocks from Peer [{peer}]",
-                            peer.Address.ToHex()
-                            );
+                            "Got all blocks from Peer [{{peer}}]. " +
+                            "(Count: {{count}}, TipIndex: #{{index}}, TipHash: {{hash}})",
+                            peer.Address.ToHex(),
+                            receivedBlockCountCurrentLoop,
+                            workspace.Tip.Index,
+                            workspace.Tip.Hash
+                        );
```

Comment:
In my thought, it will be represented like `Got all blocks from Peer [{000000...}]. (Count: {0}, TipIndex: #{0}, TipHash: {000000...})` through the example of [Serilog site](https://serilog.net/). I left this comment because it seems you intended `$"Got all blocks from Peer [{{peer}}]",` as [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) syntax. Or does it work in another way?

Reply:
It is Serilog message template syntax. [ref.](https://github.com/serilog/serilog/wiki/Writing-Log-Events#message-template-syntax) 
`{{` will be rendered as `{` and property names are written between `{` and `}` brackets.

Reply:
Okay 👌 , As you described, if `{{` will be rendered as `{`, will not they be rendered as `Got all blocks from Peer [{peer}]. (Count: {count}, TipIndex: #{index}, TipHash: {hash})`?

I tested it with below test code:

```csharp
using (var log = new LoggerConfiguration()
    .WriteTo.Console()
    .MinimumLevel.Debug()
    .CreateLogger())
{
    log.Debug(
        "Got all blocks from Peer [{{peer}}]. " +
        "(Count: {{count}}, TipIndex: #{{index}}, TipHash: {{hash}})",
        new Address().ToHex(),
        0,
        0,
        new HashDigest<SHA256>()
    );
}
```

and I could saw it
```
[18:27:09 DBG] Got all blocks from Peer [{peer}]. (Count: {count}, TipIndex: #{i
ndex}, TipHash: {hash})
```

Reply:
My fault. The property name is should be started at uppercase. I did fix this problem at https://github.com/planetarium/libplanet/pull/1115/commits/40c195d5d4220cf36493dfcf87ad54bbaf607aec